Stripe/PublicHeaders/STPAPIResponseDecodable.h

```diff
@@ -11,6 +11,11 @@
 @protocol STPAPIResponseDecodable <NSObject>
 
 /**
+ A string representing the object's type. May be nil.
+ */
++ (nullable NSString *)object;
```

Comment:
Good call on renaming this to something more specific – maybe `stripeObject`?

I've added this because we now need to be able to parse a heterogenous response (adding a source to a customer can return either a token or source). The `parseResponse` method in `STPAPIRequest.m` now takes an array of serializers [0], and uses `object` to decide which one to use.

[0] This is actually misnamed, these are technically deserializers – I should rename the methods